ansible/files/airflow/airflow.cfg

```diff
@@ -89,6 +89,10 @@ authenticate = False
 # Filter the list of dags by owner name (requires authentication to be enabled)
 filter_by_owner = False
 
+# Authentication
+authenticate = {AIRFLOW_ENABLE_AUTH}
+auth_backend = airflow.contrib.auth.backends.password_auth
```

Comment:
This will require a manual step of creating a user account. We should add that to the `README`.